konrad/core.py

```diff
@@ -265,7 +295,13 @@ def run(self):
                 logger.info(f'Enter iteration {self.niter}.')
                 if self.timestep_adjuster is not None:
                     logger.info(f'Model timestep: {self.timestep}.')
-
+            
+            # Save the old radiative imbalance at the TOA. It is compared with
+            # values after adjustment to check if the model has converged.
+            
+            if self.niter != 0:
+             self.oldN = self.radiation["toa"][-1]
```

Comment:
Please indent with four spaces :)

Reply:
I need to train more on the coding best practices and `konrad` programming guidelines. Sorry. :)

Reply:
Did I ever recommend the [Black code formatter](https://github.com/psf/black) to you? It is a neat tool that does all the formatting for you (PEP8 consistent and reproducible).